hris_publisher/hris.py

```diff
@@ -0,0 +1,187 @@
+import boto3
+import json
+import os
+
+from botocore.exceptions import ClientError
+from jsonschema import validate as jsonschema_validate
+from jsonschema.exceptions import ValidationError
+
+
+class HrisJSON(object):
+    """Retrieval and serialization object for the current HRIS data file."""
+    def __init__(self, boto_session):
+        self.boto_session = boto_session
+        self.from_file = False
+        self.file_name = 'hris.json'
+        self.s3_bucket = None
+        self.bucket_name = None
+
+    def load(self):
+        """Fetch the file as stream from the s3 bucket."""
+        if self.from_file is True:
```

Comment:
im guessing this is for dev only (this section that is)

Reply:
Yes file instead of bucket section is for fixture testing only.  It defaults to false in the constructor.